src/world/action/jump.c

```diff
@@ -1,15 +1,250 @@
 #include "common.h"
 
-INCLUDE_ASM(void, "world/action/jump", func_802B6000_E24040, void);
+extern void phys_adjust_cam_on_landing();
```

Comment:
Function declarations don't need `extern` before them, and they can actually move to functions.h provided these functions live in other files.